doc/backing-store-crd.md

```diff
@@ -135,23 +141,26 @@ metadata:
   finalizers:
   - noobaa.io/finalizer
   labels:
-    app: noobaa
-  name: bs
-  namespace: noobaa
+    app: <>
+  name: <>
+  namespace: <>
 spec:
   azureBlob:
     secret:
-      name: backing-store-azure-blob-bs
-      namespace: noobaa
-    targetBlobContainer: CONTAINER
+      name: <>
+      namespace: <>
+    targetBlobContainer: <>
   type: azure-blob
 ```
 
-#### PV-POOL type
+## PV-POOL type
+Creataes a StatefulSet with a PVC mounted in each pod. Each resource will connect to the NooBaa core and provide the PV filesystem storage to be used for storing encrypted chunks of data. It is possible to configure the number of pods to be used and their PV size.
+
+Each PV needs at least 17 Gibibytes to function properly.
 
-Create NooBaa resources StatefulSet with PVC mounted in each pod. Each resource will connect to the NooBaa core and provide the PV filesystem storage to be used for storing encrypted chunks of data. It is possible to configure the number of pods to be used and their PV size.
+*In the YAML, the value of `storage` should adhere to the [K8s Memory Resource Units format](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory).
 ```shell
-noobaa -n noobaa backingstore create pv-pool bs --num-volumes 3 --pv-size-gb 32 --storage-class STORAGE-CLASS-NAME
+noobaa -n noobaa backingstore create pv-pool <BACKINGSTORE NAME> --num-volumes <> --pv-size-gb <INT >= 17> --storage-class <THE SC THAT WILL BE USED FOR PV PROVISIONING>
```

Comment:
Please write before this line the definition of `SC` before using it (or just write `storage class` here).